src/notaryUtils.js

```diff
@@ -26,11 +26,15 @@ function createCaseID () {
  */
 function setupCase (params) {
   return co(function * () {
-    const uniqueID = params.caseID || uuid()
+    // the caseID param will be in the format of notary/cases/uuid,
+    // only check if its uuid is less than 40 characters
+    const uniqueID = params.caseID ? params.caseID.substring(params.caseID.indexOf('/cases/') + 7) : uuid()
```

Comment:
Hmm, it seems a little bit odd that the `caseID` is the only parameter that gets validated at all. Is there a specific reason to have that check that the unique id is the right length?